packages/jsxstyle/preact/jsxstyle-preact.js

```diff
@@ -1,5 +1,5 @@
 /** @jsx h */
-import invariant from 'invariant';
+import * as invariant from 'invariant';
```

Comment:
Since we’re only using `invariant` in one place in runtime `jsxstyle`, I think we could pretty safely replace `invariant` with a single thrown error.

Reply:
Sounds great.  Fewer dependencies (especially CJS ones) is a good thing!

Reply:
yo, I love removing dependencies.